zebra-rpc/src/methods/types/transaction.rs

```diff
@@ -649,6 +682,19 @@ impl TransactionObject {
                     value_balance_zat: tx.orchard_value_balance().orchard_amount().zatoshis(),
                 })
             },
+            binding_sig: tx.sapling_binding_sig().map(|raw_sig| raw_sig.into()),
+            joinsplit_pub_key: tx.joinsplit_pub_key().map(|raw_key| {
+                // Display order is reversed in the RPC output.
+                raw_key
+                    .as_ref()
+                    .iter()
+                    .rev()
+                    .copied()
+                    .collect::<Vec<_>>()
+                    .try_into()
+                    .unwrap()
+            }),
```

Comment:
```suggestion
            joinsplit_pub_key: tx.joinsplit_pub_key().map(|raw_key| {
                let mut key: [u8; 32] = raw_key.into();
                key.reverse();
                key
            }),
```

Optional: Can use this instead, we did it in other places, not sure if it is better but is a bit shorter.

Reply:
That's indeed better, sometimes I go too wild with the functional syntax :sweat_smile: 

Changed in c200a470c